sstables/index_reader.hh

```diff
@@ -186,6 +186,12 @@ public:
     }
 
     processing_result process_state(temporary_buffer<char>& data) {
+        if (_abort.abort_requested()) {
+            auto ex = _abort.abort_requested_exception_ptr();
+            sstlog.warn("aborting index_consume_entry_context::process_state() : {}", ex);
+            std::rethrow_exception(ex);
+        }
```

Comment:
Why the warning here? What's the user going to do about it?

I understand you wanted to see it during debugging, but now, it's just noise. You can all abort_source::check().

Reply:
I agree, I have removed the warning and updated the code.